modules/exploits/multi/http/struts_code_exec_classloader.rb

```diff
@@ -97,9 +97,13 @@ def jsp_dropper(file, exe)
 
   def dump_line(uri, cmd = "")
     res = send_request_cgi({
-      'uri'     => uri+cmd,
+      'uri'     => uri,
+      'encode_params' => false,
+      'vars_get' => {
+        cmd => ""
```

Comment:
it shoudl result it /uri?my_command_to_execute so I think the syntax is correct, we don't need a value here